Web/starter_project_vanguard/pages/blog/add-new-blog.tsx

```diff
@@ -0,0 +1,82 @@
+import FileUpload from '@/components/blog/FileUpload'
+import TextEditor from '@/components/blog/TextEditor'
+
+import React, { useState } from 'react'
+
+const addNewBlog = () => {
+  const tags = [
```

Comment:
make this tags outside of the code in data folder then import it here. because there might be change on the tags ever time.